src/sanaprotocolbuilder/urls.py

```diff
@@ -6,7 +6,9 @@
 from webapp import views as webappViews
 from rest_framework.authtoken import views as restViews
 
-urlpatterns = patterns('',
+urlpatterns = patterns(
+    '',
```

Comment:
Let's keep the prefixes on the same line as `patterns(` as a Django convention (to keep it consistent with the docs)